Src/BootCamp.Chapter/CaesarCipher.cs

```diff
@@ -11,12 +7,12 @@ public static class CaesarCipher
     {
         public static string Encrypt(string message, byte shift)
         {
-            return message;
+            return Lesson10.Cypher(message, shift);
```

Comment:
Please move the implementation here. It is redundant like this.

Reply:
I moved the Cypher function to the same class